typescript/web-app/package.json

```diff
@@ -18,43 +18,44 @@
 	"license": "MIT",
 	"dependencies": {
 		"@azure/storage-blob": "12.12.0",
-		"@itwin/appui-abstract": "^3.5.0",
-		"@itwin/appui-layout-react": "^3.5.0",
-		"@itwin/appui-react": "^3.5.0",
+		"@itwin/appui-abstract": "^3.7.8",
+		"@itwin/appui-layout-react": "^3.7.8",
+		"@itwin/appui-react": "^3.7.8",
 		"@itwin/browser-authorization": "^0.5.1",
-		"@itwin/components-react": "^3.5.0",
-		"@itwin/core-bentley": "^3.5.0",
-		"@itwin/core-common": "^3.5.0",
-		"@itwin/core-frontend": "^3.5.0",
-		"@itwin/core-geometry": "^3.5.0",
-		"@itwin/core-markup": "^3.5.0",
-		"@itwin/core-orbitgt": "^3.5.0",
-		"@itwin/core-quantity": "^3.5.0",
-		"@itwin/core-react": "^3.5.0",
-		"@itwin/imodel-components-react": "^3.5.0",
+		"@itwin/components-react": "^3.7.8",
+		"@itwin/core-bentley": "^3.7.8",
+		"@itwin/core-common": "^3.7.8",
+		"@itwin/core-frontend": "^3.7.8",
+		"@itwin/core-geometry": "^3.7.8",
+		"@itwin/core-markup": "^3.7.8",
+		"@itwin/core-orbitgt": "^3.7.8",
+		"@itwin/core-quantity": "^3.7.8",
+		"@itwin/core-react": "^3.7.8",
+		"@itwin/imodel-components-react": "^3.7.8",
 		"@itwin/imodels-access-frontend": "^1.0.1",
 		"@itwin/imodels-client-management": "^1.0.1",
 		"@itwin/itwinui-icons-react": "^1.16.0",
 		"@itwin/itwinui-react": "^1.39.0",
-		"@itwin/presentation-common": "^3.5.0",
-		"@itwin/presentation-components": "^3.5.0",
-		"@itwin/presentation-frontend": "^3.5.0",
+		"@itwin/presentation-common": "^3.7.8",
+		"@itwin/presentation-components": "^3.7.8",
+		"@itwin/presentation-frontend": "^3.7.8",
 		"@itwin/reality-data-client": "^0.9.0",
 		"@itwin/web-viewer-react": "3.0.8",
-		"@itwin/webgl-compatibility": "^3.5.0",
+		"@itwin/webgl-compatibility": "^3.7.8",
 		"buffer": "^6.0.3",
 		"fast-png": "^6.1.0",
 		"jshint": "^2.13.1",
 		"react": "^17.0.0",
 		"react-dom": "^17.0.0",
 		"react-redux": "^7.2.0",
-		"reality-capture": "file:../sdk"
+		"@itwin/reality-capture": "workspace:^*"
 	},
 	"devDependencies": {
 		"@bentley/react-scripts": "^5.0.0",
-		"@itwin/build-tools": "^3.5.0",
+		"@itwin/build-tools": "^3.7.8",
 		"@types/cors": "^2.8.12",
 		"@types/node": "^16.11.34",
+		"@types/react": "17.0.0",
 		"@types/react-dom": "^16.9.0",
 		"@typescript-eslint/eslint-plugin": "^5.30.6",
 		"@typescript-eslint/parser": "^5.30.6",
```

Comment:
where is the dep on webpack-cli on L66 being used?

Reply:
removed the unnecessary dependency